sdks/java/io/google-cloud-platform/build.gradle

```diff
@@ -146,6 +147,7 @@ dependencies {
   testImplementation library.java.powermock
   testImplementation library.java.powermock_mockito
   testImplementation library.java.joda_time
+  testImplementation "com.google.cloud:google-cloud-spanner:6.17.4:tests"
```

Comment:
Can the version come from the BOM? (if you remove the version, does it work?)

Could you create a variable as the other libraries (e.g. `library.java.spanner.tests`)?

Reply:
It doesn't work if I remove the version, but I've moved this to `BeamModulePlugin.groovy` with the versions defined by variables